specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-12-10-preview/examples/HostPool_List.json

```diff
@@ -36,7 +36,7 @@
           },
           {
             "name": "hostPool2",
-            "id": "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourceGroups/resourceGroup1/providers/Microsoft.DesktopVirtualization/hostPools/hostPool2",
+            "id": "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourceGroups/MyResourceGroup/providers/Microsoft.DesktopVirtualization/hostPools/hostPool2",

```

Comment:
should this be MyHostPool2 then? Applies to all _List?

Reply:
updated here: https://github.com/Azure/azure-rest-api-specs/pull/9766/commits/ef594c02297131ec5e9cd957f4ec0cfe0efcaf07